src/api.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export const getAllReviews = () => {
+  return axios.get("https://nicos-nc-games.onrender.com/api/reviews");
```

Comment:
I would take the data off the .data.reviews keys here rather than in the useEffect 🕵️